src/services/carrier_account_service.js

```diff
@@ -71,20 +71,20 @@ export default (easypostClient) =>
      * @param {string} carrierAccountType
      * @returns {string}
      */
-    static selectCarrierAccountCreationEndpoint(carrierAccountType) {
+    static _selectCarrierAccountCreationEndpoint(carrierAccountType) {
       if (Constants.CARRIER_ACCOUNTS_WITH_CUSTOM_WORKFLOWS.includes(carrierAccountType)) {
         return 'carrier_accounts/register';
       }
       return 'carrier_accounts';
     }
 
     /**
-     * Retrieve a list of all carrier accounts associated with the API key.
+     * Retrieve a list of all carrier accounts associated with the API #.
```

Comment:
```suggestion
     * Retrieve a list of all carrier accounts associated with the API key.
```

Reply:
Ooo, nice catch! This was unintended, search and replace gone wrong 😅